build/build-darwin.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
```

Comment:
Yea, I haven't perused it too much either. However, I think there are ways to perform builds without needing to run every OS -- there would be a lot less Electron apps if that were the case =/